util/sock.c

```diff
@@ -34,6 +34,8 @@ sock_resolve_unix(const char * addr)
 	if ((sa_un = calloc(1, sizeof(struct sockaddr_un))) == NULL)
 		goto err0;
 	sa_un->sun_family = AF_UNIX;
+
+	/* Safely copy addr into the structure. */
```

Comment:
I keep on getting complaints about the `strcpy()` from static analysis programs, so having the "safely copy" in its own "paragraph" would be helpful in quickly recognizing that it's a false positive.